### PR TITLE
New 8051 cheat sheet

### PR DESCRIPTION
**Pull request by mRse7enNIT**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {8051} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**

{Short Description}`**

Provides basic instruction sets of 8051 uC.
---
***********************************************************************************************
Not able to create instant answer page.. the webpage is not creating one.
Instant Answer Page: https://duck.co/ia/view/{{ID}}

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mRse7enNIT** [comment]
when I am clicking 'Instant answer page' its not proceeding




On Sun, May 22, 2016 at 10:29 PM, Dax <notifications@github.com> wrote:

> Metadata placeholder post
>
> *If you haven't already, please edit your first comment to include a link
> to the IA page at the bottom.*
> ------------------------------
>
> *How to find your IA page*
>
>    -
>
>    New Instant Answer
>    -- follow the steps here <https://duck.co/ia/new_ia> to create your IA
>    page.
>    -
>
>    Existing Instant Answer
>    -- find your IA page from this list <https://duck.co/ia>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3120#issuecomment-220843268>
>



-- 
SAPTARSHI MITRA

Undergraduate Student (ROLL - 13/EC/23)
Department of Electronics and Communication Engineering
National Institute of Technology, Durgapur,
West Bengal-713 209

email : saptarshi14mitra@gmail.com
phone : +91 9038263472

File: share/goodie/cheat_sheets/json/8051.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+    "template_type": "reference",
+
+    "section_order": [
+        "Instructions"
+        
+    ],
+
+    "sections": {
+        "Intructions": [
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/8051.json]: We should split this up into smaller sections, either `Instructions (N)` or some more specific categories.

**Comment by tagawa** [comment]
Hi again @mRse7enNIT. Thanks for creating this and sorry you've had some problems.

Regarding the Instant Answer page, have you tried using this page? https://duck.co/ia/new_ia
If so, what happened when you submitted the form?

Regarding this pull request, it looks like a separate file (multisim.json) is also included, which has created merge conflicts. There's a link above called "command line instructions" which shows how it can be fixed from the command line. Another way is to create a new PR. If you do this, please first create a new Git branch, e.g. `git checkout -b 8051-cheat-sheet`. Then, make sure your 8051 file is the only file that is commited and pushed. You can check this using `git status`.

I hope this helps - let me know if you have problems or ask for support in the #ask-anything Slack channel.

**Comment by mRse7enNIT** [comment]
Hello,

I pushed from a different branch . But the IA page submission i not
working. When i am clicking it it is freezed in a same page.. So, can't
generate the IA page.




On Mon, May 23, 2016 at 9:02 AM, Daniel Davis <notifications@github.com>
wrote:

> Hi again @mRse7enNIT <https://github.com/mRse7enNIT>. Thanks for creating
> this and sorry you've had some problems.
>
> Regarding the Instant Answer page, have you tried using this page?
> https://duck.co/ia/new_ia
> If so, what happened when you submitted the form?
>
> Regarding this pull request, it looks like a separate file (multisim.json)
> is also included, which has created merge conflicts. There's a link above
> called "command line instructions" which shows how it can be fixed from the
> command line. Another way is to create a new PR. If you do this, please
> first create a new Git branch, e.g. git checkout -b 8051-cheat-sheet.
> Then, make sure your 8051 file is the only file that is commited and
> pushed. You can check this using git status.
>
> I hope this helps - let me know if you have problems or ask for support in
> the #ask-anything Slack channel.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3120#issuecomment-220880785>
>



-- 
SAPTARSHI MITRA

Undergraduate Student (ROLL - 13/EC/23)
Department of Electronics and Communication Engineering
National Institute of Technology, Durgapur,
West Bengal-713 209

email : saptarshi14mitra@gmail.com
phone : +91 9038263472

**Comment by tagawa** [comment]
@mRse7enNIT Thanks very much for trying to fix this. The other branch worked so I added a few comments to that.

I'm not sure what's happened with creating the IA page. I've actually had a problem trying to create it as well. I'll look into this and will let you know when it's OK.

**Pull request closed without merging**